### PR TITLE
Device config

### PR DESCRIPTION
**Pull request by rajsing**
Changes to take params from pipeline

### CODE REVIEW

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public class GetAzureStorSimpleJob : StorSimpleCmdletBase
         /// </summary>
         [Parameter(Mandatory=true, Position = 0, ParameterSetName = StorSimpleCmdletParameterSet.IdentifyById,
             HelpMessage = StorSimpleCmdletHelpMessage.DeviceJobId)]
-        public string JobId { get; set; }
+        public string InstanceId { get; set; }
```
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs]: ValidateNotNullOrEmpty

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs [hunk 2]
```diff
@@ -122,7 +122,7 @@ public override void ExecuteCmdlet()
                 }
                 else
                 {
-                    WriteObject(response.DeviceJobList);
+                    WriteObject(response.DeviceJobList, true);
```
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs]: what is the significance of this additional argument?

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx [hunk 3]
```diff
@@ -430,7 +430,7 @@
     <value>Data0 configuration with Controller0 and Controller1 IP Addresses along with TimeZone and Primary DNS Server information is mandatory when configuring the device for the first time</value>
   </data>
   <data name="DeviceJobsNextPageFormatMessage" xml:space="preserve">
-    <value>More jobs are available for your query. To access the next page of your result use \"-First {0} -Skip {1}\" in your commandlet</value>
+    <value>More jobs are available for your query. To access the next page of your result use -First {0} -Skip {1} in your commandlet</value>
   </data>
   <data name="DeviceJobsNextPagewithNoFirstMessage" xml:space="preserve">
     <value>More jobs are available in the subsequent pages for your query. To access the next page use \"-Skip {0}\"  in your commandlet</value>
```
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: Remove \" from this message as well

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs (hunk 1 above)
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs]: Make sure you change the user message in HandleDeviceJobResonse.

Comment (rajsing) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs]: I didnt get this one. Where is HandlerDeviceJobResponse?

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs (hunk 2 above)
Comment (rajsing) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs]: The additional argument enumerates the object. Without that it comes as an unenumerated list

So @(Get-AzureStorSimpleJob -DeviceName $deviceName) will end up being an array with a single element which is a list while it should ideally be an array of the elements in the list.

The same has been done in Get-AzureStorSimpleDeviceBackup

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs (hunk 1 above)
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs]: It is in StorSimpleCmdletBase.cs. Basically it prints out a message to the user to use Get-JobStatus -JobId {0} to get the device job status. You need to changes this message in Resources.resx file.

**Pull request merged**